graviton/blackbox.py

```diff
@@ -267,7 +269,8 @@ class DryRunEncoder:
     def encode_args(
         cls,
         args: Sequence[PyTypes],
-        abi_types: Optional[List[Optional[abi.ABIType]]] = None,
+        abi_types: Optional[List[EncodingType]] = None,
+        validation: bool = True,
```

Comment:
@tzaffi Minor:  `encode_args` docstring does _not_ describe the added parameter (`validation`).  I raise the note because the docstring covers existing parameters.

Reply:
Addressed: 5ac666e9415096e2969be57e0e4783ccf42759ee